addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/data/TankerkoenigService.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.tankerkoenig.internal.data;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Properties;
+
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.openhab.binding.tankerkoenig.internal.config.OpeningTimes;
+import org.openhab.binding.tankerkoenig.internal.config.TankerkoenigDetailResult;
+import org.openhab.binding.tankerkoenig.internal.config.TankerkoenigListResult;
+import org.openhab.binding.tankerkoenig.internal.serializer.CustomTankerkoenigDetailResultDeserializer;
+import org.openhab.binding.tankerkoenig.internal.serializer.CustomTankerkoenigListResultDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/***
+ * Serivce class requesting data from tankerkoenig api and providing result objects
+ *
+ * @author Dennis Dollinger
+ * @author Juergen Baginski
+ */
+public class TankerkoenigService {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    private final GsonBuilder GSON_BUILDER = new GsonBuilder().registerTypeAdapter(TankerkoenigListResult.class,
```

Comment:
Please make all of them lower case since they are not constants anymore but just final fields